src/components/page-scheduled-job/index.js

```diff
@@ -85,9 +85,9 @@ const PageScheduledJob = (props) => {
                   </Typography>
                 )}
               {scheduledJob?.message && (
-                <Code>
-                  <strong>{scheduledJob.message}</strong>
-                </Code>
+                <strong>
+                  <Code>{scheduledJob.message}</Code>
+                </strong>
```

Comment:
The text used to be encapsulated in `<strong>` before the changes. Since the input to the component is string, the tag was moved to encapsulate the component instead.

Reply:
However I'm not sure if it's necessary

Reply:
updated